src/app/lib/layoutGrid.stories.jsx

```diff
@@ -0,0 +1,194 @@
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '11' }}>
+          <ColouredDiv colour={green}>6/6 or 11/12</ColouredDiv>
+        </NestedGridItemLarge>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '12' }}>
+          <ColouredDiv colour={blue}>6/6 or 12/12</ColouredDiv>
+        </NestedGridItemLarge>
+      </GridItemConstrainedLarge>
+    </GhostWrapper>
+  ))
+  .add('GridItemConstrainedLargeNoMargin', () => (
+    <GhostWrapper>
+      <GridItemConstrainedLargeNoMargin>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '1' }}>
+          <ColouredDiv colour={red}>1/6 or 1/12</ColouredDiv>
+        </NestedGridItemLarge>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '2' }}>
+          <ColouredDiv colour={orange}>2/6 or 2/12</ColouredDiv>
+        </NestedGridItemLarge>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '3' }}>
+          <ColouredDiv colour={yellow}>3/6 or 3/12</ColouredDiv>
+        </NestedGridItemLarge>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '4' }}>
+          <ColouredDiv colour={green}>4/6 or 4/12</ColouredDiv>
+        </NestedGridItemLarge>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '5' }}>
+          <ColouredDiv colour={blue}>5/6 or 5/12</ColouredDiv>
+        </NestedGridItemLarge>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '6' }}>
+          <ColouredDiv colour={violet}>6/6 or 6/12</ColouredDiv>
+        </NestedGridItemLarge>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '7' }}>
+          <ColouredDiv colour={indigo}>6/6 or 7/12</ColouredDiv>
+        </NestedGridItemLarge>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '8' }}>
+          <ColouredDiv colour={red}>6/6 or 8/12</ColouredDiv>
+        </NestedGridItemLarge>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '9' }}>
+          <ColouredDiv colour={orange}>6/6 or 9/12</ColouredDiv>
+        </NestedGridItemLarge>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '10' }}>
+          <ColouredDiv colour={yellow}>6/6 or 10/12</ColouredDiv>
+        </NestedGridItemLarge>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '11' }}>
+          <ColouredDiv colour={green}>6/6 or 11/12</ColouredDiv>
+        </NestedGridItemLarge>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '12' }}>
+          <ColouredDiv colour={blue}>6/6 or 12/12</ColouredDiv>
+        </NestedGridItemLarge>
+      </GridItemConstrainedLargeNoMargin>
+    </GhostWrapper>
+  ))
+  .add('Example of multi-breakpoint span', () => (
+    <GhostWrapper>
+      <GridItemConstrainedLargeNoMargin>
+        <NestedGridItemLarge gridColumnStart="1" gridSpan={{ default: '12' }}>
+          <ColouredDiv colour={blue}>AN IMAGE OF YORK RUNTE</ColouredDiv>
```

Comment:
This suggestion is just to inform those looking at the Storybook example, the behaviour they should be seeing. 
```suggestion
          <ColouredDiv colour={blue}>Spanning 12 columns - e.g. and image</ColouredDiv>
```